packages/runtime/container-runtime/.eslintrc.cjs

```diff
@@ -83,9 +83,14 @@ module.exports = {
 			rules: {
 				// TODO: remove these overrides and fix violations
 				"@typescript-eslint/explicit-function-return-type": "off",
+				"unicorn/error-message": "warn",
```

Comment:
Left as a warning for now. Didn't want to invent error messages that may/may not make sense.